cli/create.ts

```diff
@@ -17,16 +17,18 @@ interface CreateArgs {
   codaApiEndpoint: string;
   name?: string;
   description?: string;
+  workspaceId?: string;
```

Comment:
Should we allow users to pass a url or an id, and if it's a url we parse out the workspace id? Small things like this help a bit with usability because users often don't know how to get an id, but are generally comfortable copy/pasting urls from the browser if they know which page.

Also I think coda.ts needs to be updated to define the new flag. We could call it `workspace` so it's a little clearer you can pass a url or an id.

Reply:
yeah good callout, let me update!